app/controllers/sessions_controller.rb

```diff
@@ -1,19 +1,22 @@
 class SessionsController < ApplicationController
   def new
-    redirect_to home_path if current_user  
+    redirect_to home_path if current_user
   end
-  
+
   def create
     user = User.where(email: params[:email]).first
-    if user && user.authenticate(params[:password])
+    if user && user.authenticate(params[:password]) && user.active
```

Comment:
I think you can use `user.active?` - boolean fields in Rails receive automatically question-mark getters as well :)